split_train_val_test.py

```diff
@@ -0,0 +1,65 @@
+import os
+import os.path as osp
+import shutil
+import random
+import argparse
+
+if __name__ == '__main__':
+    PARSER = argparse.ArgumentParser()
+    PARSER.add_argument("--data-dir", type=str,
+                        default="/data/jiachen.li/iu_xray/ecgen-radiology/",
```

Comment:
Let's not add your personal path here.

Reply:
Sounds great!

Reply:
You actually still have your personal files here. Actually, there is no need to set a default because you don't know how other people's file system looks like

Reply:
Oh sorry about that. I misidentified it as another file. Now I have searched my name globally to make sure my name will not appear in all files